x-pack/plugins/cases/public/files/index.ts

```diff
@@ -17,6 +17,12 @@ const buildFileKind = (config: FilesConfig, owner: Owner): FileKindBrowser => {
     id: constructFileKindIdByOwner(owner),
     allowedMimeTypes: config.allowedMimeTypes,
     maxSizeBytes: config.maxSize ?? MAX_FILE_SIZE,
+    managementUiActions: {
+      delete: {
+        enabled: false,
+        reason: 'File is managed by Cases',
```

Comment:
```suggestion
        reason: 'This file is managed by Cases. Navigate to the Cases page under Stack management to delete it.',
```

Reply:
I talked to @szabosteve offline and since the file might exist in a different plugin he suggested something like this:

```
This file is managed by Cases. Navigate to the Cases page under <Security or Observability or Stack Management> to delete it.
```